src/utils/emoji-index/nimble-emoji-index.js

```diff
@@ -33,7 +35,16 @@ export default class NimbleEmojiIndex {
         })
       }
 
-      this.emojis[id] = getSanitizedData(id, null, null, this.data)
+      // If skin variations include them
+      if (skin_variations) {
+        this.emojis[id] = {}
+        for (let skinTone = 1; skinTone <= 6; skinTone++) {
+          this.emojis[id][skinTone] = getSanitizedData({id, skin: skinTone}, skinTone, this.set, this.data)
+        }
```

Comment:
This could probably be smaller if it were an array, but an object of IDs 1-6 is fine